### PR TITLE
Provide implementation for localFS

### PR DESCRIPTION
**Pull request by lbcjones**

### CODE REVIEW

File: gcpFS/FileOps.go [hunk 1]
```diff
@@ -10,58 +10,47 @@ import (
...
 	ctx := context.Background()
 	client, err := storage.NewClient(ctx)
 	if err != nil {
 		return err
 	}
-	g.client = client
 	g.ctx = ctx
-	defer client.Close()
+	g.client = client
+	client.Close()
```
Comment (ninjamarcus) [inline gcpFS/FileOps.go:49]: Should this not be a defer client.Close() as opposed to a close, as won't this just shutdown the connection? Im happy to be wrong here. The examples in googles docs seem to suggest that is the way to go - https://cloud.google.com/storage/docs/creating-buckets#permissions-client-libraries

**Review by ninjamarcus — COMMENTED** [review]
Is David happy with these changes also, or will this cause his tests to fail?

File: gcpFS/FileOps.go (hunk 1 above)
Comment (lbcjones) [reply gcpFS/FileOps.go:49]: a deferred call is "deferred" until immediately before the enclosing function returns. As this is already the last statement in said function, there is no point in deferring it.

Comment (lbcjones) [reply gcpFS/FileOps.go:49]: I suspect that the client.Close() is actually redundant here. It looks like the client will (re)establish the connection when it is actually used. Also, according to the docs, it is not "necessary" to call close prior to the program exiting.

**Comment by lbcjones** [comment]
> Is David happy with these changes also, or will this cause his tests to fail?
David is happy in principle, although yes I need to make suitable changes in lb_central once the PR has been merged.

**Pull request merged**